src/merger_tree.cpp

```diff
@@ -0,0 +1,175 @@
+#include <vector>
+#include <cassert>
+
+#include "mpi_wrapper.h"
+#include "merger_tree.h"
+#include "datatypes.h"
+#include "subhalo.h"
+#include "locate_ids.h"
+#include "argsort.h"
+#include "reorder.h"
```

Comment:
Is this needed here? I cannot see it being called in this file.

Reply:
Probably left over from an earlier iteration. I'll remove it.

Reply:
Removed.